libs/sdk-ui-kit/src/utils/useId.ts

```diff
@@ -0,0 +1,8 @@
+// (C) 2025 GoodData Corporation
+import { useState } from "react";
+import { v4 as uuid } from "uuid";
+
+export const useId = (): string => {
```

Comment:
Minor suggestion: Maybe put a note about React 18 and that it can be replaced with native `useId` once this is the oldest supported version by SDK.

Reply:
added